src/e2e-test/features/servicenowsource/RunTime.feature

```diff
@@ -0,0 +1,70 @@
+    And Select plugin: "ServiceNow" from the plugins list as: "Source"
+    And Navigate to the properties page of plugin: "ServiceNow"
+    And configure ServiceNow source plugin for table: "RECEIVING_SLIP_LINE" in the Table mode
+    And fill Credentials section for pipeline user
+    And Enter input plugin property: "startDate" with value: "start.date"
+    And Enter input plugin property: "endDate" with value: "end.date"
+    Then Validate "ServiceNow" plugin properties
+    And Capture the generated Output Schema
+    And Close the Plugin Properties page
+    And Select Sink plugin: "BigQueryTable" from the plugins list
+    And Connect source as "ServiceNow" and sink as "BigQuery" to establish connection
+    And Navigate to the properties page of plugin: "BigQuery"
+    And Replace input plugin property: "project" with value: "projectId"
+    And Enter input plugin property: "datasetProject" with value: "datasetprojectId"
+    And Configure BigQuery sink plugin for Dataset and Table
+    Then Validate "BigQuery" plugin properties
+    And Close the Plugin Properties page
+    And Preview and run the pipeline
+    Then Verify the preview of pipeline is "success"
+    And Click on the Preview Data link on the Sink plugin node: "BigQueryTable"
+    Then Verify sink plugin's Preview Data for Input Records table and the Input Schema matches the Output Schema of Source plugin
+
+  @TS-SN-RNTM-2 @SN_SOURCE_CONFIG @SN_RECEIVING_SLIP_LINE @BQ_SINK
+  Scenario: Verify user should be able to deploy and run the pipeline where ServiceNow source is configured for Table mode
+    When Open Datafusion Project to configure pipeline
+    And Select plugin: "ServiceNow" from the plugins list as: "Source"
+    And Navigate to the properties page of plugin: "ServiceNow"
+    And configure ServiceNow source plugin for table: "RECEIVING_SLIP_LINE" in the Table mode
+    And fill Credentials section for pipeline user
+    And Enter input plugin property: "startDate" with value: "start.date"
+    And Enter input plugin property: "endDate" with value: "end.date"
+    Then Validate "ServiceNow" plugin properties
+    And Close the Plugin Properties page
+    And Select Sink plugin: "BigQueryTable" from the plugins list
+    And Connect source as "ServiceNow" and sink as "BigQuery" to establish connection
+    And Navigate to the properties page of plugin: "BigQuery"
+    And Replace input plugin property: "project" with value: "projectId"
+    And Enter input plugin property: "datasetProject" with value: "datasetprojectId"
+    And Configure BigQuery sink plugin for Dataset and Table
+    Then Validate "BigQuery" plugin properties
+    And Close the Plugin Properties page
+    And Save and Deploy Pipeline
+    And Run the Pipeline in Runtime
+    And Wait till pipeline is in running state
+    And Open and capture logs
+    And Verify the pipeline status is "Succeeded"
+    And Verify count of no of records transferred to the target BigQuery Table
```

Comment:
in addition to count, verify that correct data is written to sink

Reply:
Same reason as above, I will raise a Separate PR for this.